markdown/3.0/en/documentation/include-scripts.md

```diff
@@ -4,31 +4,27 @@
 
 The following section shows a boilerplate for including FACT-Finder Web Components in your website. `custom-elements-es5-adapter.js` and `webcomponents-loader.js` will load the required Polyfills minimized to the needs of the calling browser. 
 
+Note that the [default-styles.css](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/release/3.0/dist/default-styles.css) within our dist folder is already bundled into `bundle.js` and hence is not required to be referenced on your page. It is only for reference.
+
 **Boilerplate Code**
 ```html
 <head>
-    <meta charset="UTF-8">
-    <title>Page Title</title>
-
-    <!-- Do not change the order of the scripts -->
+    <!-- Do not change the order of the scripts to ensure all required polyfills are loaded before our script -->
     <script src="../dist/vendor/custom-elements-es5-adapter.js"></script>
     <script src="../dist/vendor/webcomponents-loader.js"></script>
     <script defer src="../dist/bundle.js"></script>
-    
-    <!-- You are free to customize those styles to your needs -->
-    <link rel="stylesheet" type="text/css" href="default-styles.css" />
 </head>
 ```
 
 ## Prevent Flash Of Unstyled Content (FOUC)
 
 ---
 
-In browsers where Web Components are not natively supported you might encounter ugly flashing of unstyled content while the page is loading. To prevent this just annotate all elements that have a visual component and are shown immediately on page load with the `unresolved` attribute.
+In browsers where Web Components are not natively supported you might encounter ugly flashing of unstyled content while the page is loading. To prevent this just annotate all elements that have a visual component and are shown immediately on page-load with the `unresolved` attribute.
```

Comment:
```suggestion
Depending on the browser's page load time you might encounter ugly flashes of unstyled content and uncomplete HTML template pieces. To prevent this just annotate all elements that have a visual component and are shown immediately on page-load with the `unresolved` attribute.
```

Reply:
```suggestion
Depending on the browser's page load time you might encounter ugly flashes of unstyled content and incomplete HTML template pieces. To prevent this simply annotate all elements that have a visual component and are shown immediately after page load with the `unresolved` attribute.
```